config.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+return [
+    'metabox' => [
+        'id' => 'service_booking_settings',
+        'post_type' => 'post', //'download',
+        'title' => __('Booking Options', 'eddbk'),
+    ],
+    'menu' => [
+        'root' => [
+            'page_title' => __('Bookings', 'eddbk'),
+            'menu_title' => __('Bookings', 'eddbk'),
+            'capability' => 'publish_posts', //publish_bookings
+            'menu_slug' => 'eddbk-bookings',
+            'icon' => 'dashicons-calendar-alt',
+            'position' => 20
+        ],
+        'settings' => [
+            'page_title' => __('Settings', 'eddbk'),

```

Comment:
Data in the config must be static. This assumes the existence of the `__()` function, and invokes it, which should never happen. On the other hand, such strings definitely need to be i18n-ized. @mecha, what do you think we can do about this?